knowledge/data-sources/website/main.go

```diff
@@ -63,15 +64,15 @@ func main() {
 
 	inputData := os.Getenv("GPTSCRIPT_INPUT")
 	input := MetadataInput{}
-	for i := range input.WebsiteCrawlingConfig.URLs {
```

Comment:
This was put in to fix an issue where spaces in the URL caused errors. Has this been addressed somewhere else?

Reply:
Yeah, i addressed this in client side. It is better to address in client side because we have other checks that adds protocol automatically.